src/api/ledger/accountinfo.js

```diff
@@ -6,7 +6,43 @@ const removeUndefined = require('./parse/utils').removeUndefined;
 const validate = utils.common.validate;
 const composeAsync = utils.common.composeAsync;
 
-function formatAccountInfo(response) {
+type AccountData = {
+  Sequence: number,
+  Account: string,
+  Balance: string,
+  Flags: number,
+  LedgerEntryType: string,
+  OwnerCount: number,
+  PreviousTxnID: string,
+  AccountTxnID: string,
```

Comment:
I have no idea how the flow typechecking system works, as I've no experience with, but I can say AccountTxnID may not always be present. 

I saw some `flow` types, with names suffixed with `?`, which I  assumed meant optional?